class.jetpack-gutenberg.php

```diff
@@ -155,109 +164,31 @@ static function load( $response = null, $handler = null, $request = null ) {
 		 *
 		 * @param boolean
 		 */
-		if ( apply_filters( 'jetpack_load_beta_blocks', $is_availability_endpoint_beta ) ) {
+		if ( apply_filters( 'jetpack_load_beta_blocks', false ) ) {
 			Jetpack_Constants::set_constant( 'JETPACK_BETA_BLOCKS', true );
 		}
 
 		/**
-		 * Filter the list of block editor blocks that are available through jetpack.
-		 *
-		 * This filter is populated by Jetpack_Gutenberg::jetpack_set_available_blocks
+		 * Filter the whitelist of block editor extensions that are available through Jetpack.
 		 *
 		 * @since 6.8.0
 		 *
 		 * @param array
 		 */
-		self::$blocks = apply_filters( 'jetpack_set_available_blocks', self::$default_blocks );
-
-		/**
-		 * Filter the list of block editor plugins that are available through jetpack.
-		 *
-		 * This filter is populated by Jetpack_Gutenberg::jetpack_set_available_blocks
-		 *
-		 * @since 6.9.0
-		 *
-		 * @param array
-		 */
-		self::$plugins = apply_filters( 'jetpack_set_available_plugins', self::$default_plugins );
```

Comment:
Seems like we're removing this filter completely. Instead, it should be deprecated instead of removed - it was out in 1 or more versions, so someone could be relying on it.

Reply:
Renamed the still-existing `jetpack_set_available_plugins` to `jetpack_set_available_extensions` (3ab63a5c98ed2ee881e72f6bcc804650cbd10c6d), and brought both `jetpack_set_available_blocks` and `jetpack_set_available_plugins` back, deprecating both (9a1abea6f901698610823ba07e341a7013da3bdf).